src/module/actor/creature/document.ts

```diff
@@ -156,6 +156,12 @@ abstract class CreaturePF2e<
         return itemTypes.spellcastingEntry.length > 0 && itemTypes.spell.length > 0;
     }
 
+    get spellcastingTraditions(): Set<string | null> {
+        return new Set(
+            this.itemTypes.spellcastingEntry.filter((e) => e.spells && e.spells.size > 0).map((e) => e.tradition),
+        );
+    }
```

Comment:
A single use during data preparation doesn't really warrant a new getter.

Reply:
Ok.  `isSpellcasting` is one, but it's used in an html template, so I suppose that is the difference.